CHANGELOG.md

```diff
@@ -2,6 +2,107 @@
+* Add Base2 prefix to internal exponential histogram classes
+  ([#5179](https://github.com/open-telemetry/opentelemetry-java/pull/5179))
+* Add MaxScale config parameter to Base2ExponentialHistogram
+  ([#5044](https://github.com/open-telemetry/opentelemetry-java/pull/5044))
+* Add close method to MetricReader
+  ([#5109](https://github.com/open-telemetry/opentelemetry-java/pull/5109))
+* Reuse AggregatorHandle with cumulative temporality to reduce allocations
+  ([#5142](https://github.com/open-telemetry/opentelemetry-java/pull/5142))
+* Delete notion of accumulation to reduce allocations
+  ([#5154](https://github.com/open-telemetry/opentelemetry-java/pull/5154))
+* Delete bound instruments
+  ([#5157](https://github.com/open-telemetry/opentelemetry-java/pull/5157))
+* Reuse aggregation handles for delta temporality to reduce allocations
+  ([#5176](https://github.com/open-telemetry/opentelemetry-java/pull/5176))
+
+#### Exporter
+
+* Add proper shutdown implementations for all exporters
+  ([#5113](https://github.com/open-telemetry/opentelemetry-java/pull/5113))
+
+#### SDK Extensions
+
+* WARNING: Remove deprecated autoconfigure exemplar filter names. Previous names `none`, `all`
+  , `with_sampled_trace` have been removed. Use `always_off`, `always_on`, `trace_based` instead.
+  ([#5098](https://github.com/open-telemetry/opentelemetry-java/pull/5098))
+* Add autoconfigure support for "none" option for propagator value
+  ([#5121](https://github.com/open-telemetry/opentelemetry-java/pull/5121))
+* Add autoconfigure support for parentbased_jaeger_remote sampler
+  ([#5123](https://github.com/open-telemetry/opentelemetry-java/pull/5123))
+* Autoconfigure closes up autoconfigured resources in case of exception
+  ([#5117](https://github.com/open-telemetry/opentelemetry-java/pull/5117))
+* View file based config has switched from snakeyaml to snakeyaml-engine for YAML parsing.
+  ([#5138](https://github.com/open-telemetry/opentelemetry-java/pull/5138))
+* Fix autoconfigure resource providers property docs
+  ([#5135](https://github.com/open-telemetry/opentelemetry-java/pull/5135))
+
+#### Testing
+
+* WARNING: Merge `opentelemetry-sdk-metrics-testing` into `opentelemetry-sdk-testing`. Stop
+  publishing `opentelemetry-sdk-metrics-testing`.
+  ([#5144](https://github.com/open-telemetry/opentelemetry-java/pull/5144))
+* Sort spans by start time (parents before children as tiebreaker) to avoid common causes for flaky
+  tests
+  ([#5026](https://github.com/open-telemetry/opentelemetry-java/pull/5026))
+* Add resource assertion methods to SpanDataAssert and MetricAssert
+  ([#5160](https://github.com/open-telemetry/opentelemetry-java/pull/5160))
+
+
+### Semantic Conventions
+
+* Update the semantic conventions to v1.17.0
+  ([#5134](https://github.com/open-telemetry/opentelemetry-java/pull/5134))
+* Update semconv to 1.18.0
+  ([#5188](https://github.com/open-telemetry/opentelemetry-java/pull/5188))
```

Comment:
```suggestion
* Update semconv to 1.18.0
  ([#5188](https://github.com/open-telemetry/opentelemetry-java/pull/5188))
```

Reply:
I'll merge these into one entry, but include links to both PRs so folks can quickly find the details of each update.